packages/core/admin/admin/src/content-manager/hooks/useLazyComponents/index.js

```diff
@@ -11,13 +11,16 @@ const componentStore = new Map();
  */
 const useLazyComponents = (componentUids = []) => {
   const [lazyComponentStore, setLazyComponentStore] = useState(Object.fromEntries(componentStore));
-  const [loading, setLoading] = useState(() => {
-    if (componentStore.size === 0 && componentUids.length > 0) {
-      return true;
-    }
+  /**
+   * Start loading only if there are any components passed in
+   * and there are some new to load
+   */
+  const newUids = useMemo(
+    () => componentUids.filter((uid) => !componentStore.get(uid)),
+    [componentUids]
```

Comment:
Just added `[componentUids]` there. If there will be a way to change loading to **true** again not from useState initial call this should be changed to `[componentUids, componentStore]` but because changing loading back to **true** not even planned in original code, just left only `[componentUids]` to get rid of eslint error. (exhaustive-deps)